src/index.html

```diff
@@ -11,7 +11,141 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="lab">Surname:
+          <input
+          type="text"
+          name="Surname"
+          minlength="2"
+          maxlength="30"
+          autocomplete="off">
```

Comment:
fix indents in all places
```suggestion
          <input
            type="text"
            name="Surname"
            minlength="2"
            maxlength="30"
            autocomplete="off"
          >
```